features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql

```diff
@@ -1564,12 +1566,14 @@ CREATE TABLE IDN_REMOTE_FETCH_REVISIONS(
 	DEPLOYED_DATE DATE,
 	DEPLOYMENT_STATUS VARCHAR2(255),
 	ITEM_NAME VARCHAR2(255),
+	DEPLOY_ERR_LOG CLOB,
 	CONSTRAINT PK_IDN_REMOTE_FETCH_REVISIONS PRIMARY KEY (ID),
 	FOREIGN KEY (CONFIG_ID) REFERENCES IDN_REMOTE_FETCH_CONFIG (ID) ON DELETE CASCADE,
 	CONSTRAINT UC_REVISIONS UNIQUE (CONFIG_ID, ITEM_NAME)
 )/
 
 
+
```

Comment:
remove extra new line

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/3143